lib/edgestore.ts

```diff
@@ -0,0 +1,11 @@
+'use client';
+
+import { type EdgeStoreRouter } from '@/app/api/edgestore/[...edgestore]/route';
+import { createEdgeStoreProvider } from '@edgestore/react';
+
+const { EdgeStoreProvider, useEdgeStore } =
+  createEdgeStoreProvider<EdgeStoreRouter>({
+    maxConcurrentUploads: 2,
```

Comment:
why ?

Reply:
just to limit the number of files loaded into EdgeStore